### ISSUE TITLE
BestBuy (US) navigation timeouts when accessed from a Canadian IP

### ISSUE THREAD

**Issue by SolidPIPe** [jef/nvidia-snatcher#501]
Hi all,

I discovered an interesting issue while trying to setup the script. I got a VPS hosted in Canada, and upon running nvidia-snatcher, I noticed that BestBuy (US) always throws a "Navigation timeout of X ms exceeded" error. At first I thought it was something with my config, given that it works fine running on my local machine in the US, but I later realized it's because I'm running a script behind a Canadian IP.

My hunch is that given that Canada has their own version of BestBuy, their IP location auto-detection routes Chromium to the Canadian BestBuy on every access, messing up with the scripts results expectation.

Attached a screenshot of the same script with the same config (.env) file running on a VPS with a Canadian IP (left) and AWS with a US IP address (right).

![CA vs US IP](https://i.imgur.com/b7FVdhz.png)

**Comment by killercode** [comment]
+1 I can't access from france :/

**Comment by jef** [comment]
This one is kind of hard to debug without `LOG_LEVEL=debug` since I don't know the request nor the response. I'm in the US, calling the US stores.

> AWS with a US IP address

I will say that _most_ stores are denying any IPs from cloud providers. So you may need to run through a US proxy.

**Comment by SolidPIPe** [comment]
I did some further testing, and now I'm almost confident that it has nothing to do with the region. I've chatted with @killercode offline and we're both running these within the OVH network. I have also tried getting an OVH VPS with a US IP address and I'm seeing the same issue with Bestbuy. It looks like Bestbuy simply blocked the entire OVH IP range. 

I tried the debug log level and got this:
![BestBuy OVH Log Level Debug](https://i.imgur.com/aEez4mR.png)
